include/eve/module/math/regular/expm1.hpp

```diff
@@ -11,73 +11,60 @@
-  //!
-  //! #### Members Functions
-  //!
-  //! | Member       | Effect                                                     |
-  //! |:-------------|:-----------------------------------------------------------|
-  //! | `operator()` | the  computation of expm1   |
-  //! | `operator[]` | Construct a conditional version of current function object |
-  //!
-  //! ---
-  //!
-  //!  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~{.cpp}
-  //!  auto operator()( floating_real_value auto x ) const noexcept;
-  //!  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-  //!
-  //! **Parameters**
-  //!
-  //!`x`:   [floating real value](@ref eve::floating_real_value).
-  //!
-  //! **Return value**
-  //!
-  //!Returns the [elementwise](@ref glossary_elementwise) exponential input minus one, with good accuracy, even for small values of `x`.
-  //!
-  //!   * If the element is \f$\pm0\f$, \f$\pm0\f$ is returned
-  //!   * If the element is \f$-\infty\f$, \f$-1\f$ is returned
-  //!   * If the element is \f$\infty\f$, \f$\infty\f$ is returned
-  //!   * If the element is a `NaN`, `NaN` is returned
-  //!
-  //! ---
-  //!
-  //!  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~{.cpp}
-  //!  auto operator[]( conditional_expression auto cond ) const noexcept;
-  //!  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-  //!
-  //!  Higher-order function generating a masked version of eve::expm1
-  //!
-  //!  **Parameters**
-  //!
-  //!  `cond` : conditional expression
-  //!
-  //!  **Return value**
-  //!
-  //!  A Callable object so that the expression `expm1[cond](x, ...)` is equivalent to `if_else(cond,expm1(x, ...),x)`
-  //!
-  //! ---
-  //!
-  //! #### Supported decorators
-  //!
-  //!  * eve::diff, eve::diff_1st, eve::diff_nth
-  //!
-  //!
-  //!     The expression `diff(expm1)(x)` computes the derivative of the function at `x`.
-  //!
-  //! #### Example
-  //!
-  //! @godbolt{doc/math/expm1.cpp}
-  //!
-  //!  @}
-  //================================================================================================
-  EVE_MAKE_CALLABLE(expm1_, expm1);
+//================================================================================================
+//! @addtogroup math_exp
+//! @{
+//! @var expm1
+//!
+//! @brief Callable object computing \f$e^x-1\f$.
+//!
+//!   **Defined in Header**
+//!
+//!   @code
+//!   #include <eve/module/math.hpp>
+//!   @endcode
+//!
+//!   @groupheader{Callable Signatures}
+//!
+//!   @code
+//!   namespace eve
+//!   {
+//!      template< eve::floating_value T >
+//!      T expm1(T x) noexcept;
+//!   }
+//!   @endcode
+//!
+//! **Parameters**
+//!
+//!   `x`:   [floating real value](@ref eve::floating_real_value).
+//!
+//! **Return value**
+//!
+//!      Returns the [elementwise](@ref glossary_elementwise) exponential input minus one, with good
+//!      accuracy, even for small values of `x`.
+//!
+//!         * If the element is \f$\pm0\f$, \f$\pm0\f$ is returned
```

Comment:
This code block is badly indented, so the whole text appears as code

Reply:
done